cms/templates/widgets/footer.html

```diff
@@ -10,37 +10,8 @@
       <div class="colophon">
         <p>&copy; ${settings.COPYRIGHT_YEAR} <a data-rel="edx.org" href="${marketing_link('ROOT')}" rel="external">${settings.PLATFORM_NAME}</a>.</p>
       </div>
-
-      <nav class="nav-peripheral">
-        <ol>
-          <li class="nav-item nav-peripheral-tos">
```

Comment:
What do you think about making these inclusion of these li's conditional on the marketing_link values being present?  And if neither is present don't render the nav at all?  If we do it that way, we could submit this change upstream.